rwengine/src/script/ScriptMachine.cpp

```diff
@@ -146,12 +146,21 @@ void ScriptMachine::executeThread(SCMThread &t, int msPassed)
         }
 
 #if RW_SCRIPT_DEBUG
-		if (strcmp(t.name, "EIGHT") == 0)
+		static auto sDebugThreadName = getenv("OPENRW_DEBUG_THREAD");
+		if (!sDebugThreadName || strncmp(t.name, sDebugThreadName, 8) == 0)
 		{
-			printf("% 8s  %04x %04x % 25s", t.name, t.programCounter, opcode, code.signature.c_str());
+			printf("%8s %01x %04x %04x %25s", t.name, t.conditionResult, t.programCounter, opcode, code.signature.c_str());
```

Comment:
25s is really short. we do have stuff like "game_character_in_vehicle" now which will break the alignment.
Just drop the alignment here. Left alignment is way easier to read anyway.